test/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ endmacro()
 
 add_subdirectory(fuzzTesting)
 add_subdirectory(libdevcore)
+add_subdirectory(libdevcore/external_dependencies)
```

Comment:
rather do a `add_subdirectory(external_dependencies)` in `libdevcore/CMakeLists.txt`